src/themes/default/lv_theme_default.c

```diff
@@ -761,13 +761,13 @@ static void theme_apply(lv_theme_t * th, lv_obj_t * obj)
 
 #if LV_USE_WIN
         /*Header*/
-        if(lv_obj_get_index(obj) == 0 && lv_obj_check_type(lv_obj_get_parent(obj), &lv_win_class)) {
+        if(lv_obj_check_type(lv_obj_get_parent(obj), &lv_win_class) && lv_obj_get_index(obj) == 0) {
```

Comment:
It can be improved further.
```
    lv_obj_t * parent;
#if LV_USE_WIN
    if (parent = lv_obj_get_parent(obj) && lv_obj_check_type(parent, &lv_win_class) && obj == lv_obj_get_child(parent, 0))
```
So there's no need to search through all children to get the index if it's negative.

Reply:
Great idea! `lv_obj_get_child` is O(1) whereas `lv_obj_get_index` is O(N).

Updated.